stravalib/client.py

```diff
@@ -32,12 +32,17 @@
 
 
 class Client(object):
-    """
-    Main client class for interacting with the exposed Strava v3 API methods.
+    """Main client class for interacting with the exposed Strava v3 API methods.
 
     This class can be instantiated without an access_token when performing authentication;
     however, most methods will require a valid access token.
 
+    Parameters
```

Comment:
i need to do a bit of digging. if the class is just an object being created we may not need these two empty items. they seem pointless.